themes/openy_themes/openy_lily/openy_lily.theme

```diff
@@ -945,3 +945,19 @@ function openy_lily_preprocess_paragraph_social_list(&$variables) {
       ->render(views_embed_view($block[0]['#name'], $block[0]['#display_id']));
   }
 }
+
+/**
+ * Implements hook_preprocess_video_embed_iframe().
+ */
+function openy_lily_preprocess_video_embed_iframe(array &$variables) {
+  if ($attr = $variables['attributes']) {
+    /** @var \Drupal\Core\Template\Attribute $attr */
+    $attr = $variables['attributes'] ?? NULL;
+    if (!$attr || !$attr instanceof Attribute) {
+      return;
+    }
+    if (!$attr->hasAttribute('frameborder')) {
```

Comment:
It looks like this condition not works... There is no border on the http://upgrade.openy.cibox.tools/build3309/about.

![Screenshot from 2021-08-03 13-55-17](https://user-images.githubusercontent.com/13733670/128004182-1ac74f14-a8e3-4e9c-b2bc-790004ca675d.png)

So on this preprocess level, we always have value for `frameborder`.

1. `frameborder` attribute hardcoded in the `renderEmbedCode` method:

- `video_embed_field/src/Plugin/video_embed_field/Provider/Vimeo.php`
- `video_embed_field/src/Plugin/video_embed_field/Provider/YouTube.php`
- `video_embed_field/src/Plugin/video_embed_field/Provider/YouTubePlaylist.php`

2. Element defined in the `video_embed_field/src/Element/VideoEmbedIFrame.php` we have there `getInfo()` method and `'#pre_render'` with `preRenderInlineFrameEmbed` function.

Proposed solution: add a custom method to the `'#pre_render'` array after `preRenderInlineFrameEmbed` function and set `frameborder` attribute to 1 inside this function. Element info can be altered with `hook_element_info_alter`, see `core/lib/Drupal/Core/Render/ElementInfoManager.php`:

```php
    // Allow modules to alter the element type defaults.
    $this->moduleHandler->alter('element_info', $info);
    $this->themeManager->alter('element_info', $info);
```

In this case, `frameborder` can be changed later in preprocess functions or in a different place.

I think we can move this to https://github.com/open-y-subprojects/openy_features/tree/main/openy_media/modules/openy_media_video.

cc @aleevas @podarok

Reply:
@hamrant thank you for your time and great advise.
I've made a PR to openy-features github  https://github.com/open-y-subprojects/openy_features/pull/8
I think this PR we can close
cc @podarok

Reply:
@podarok this PR looks ok for me - https://github.com/open-y-subprojects/openy_features/pull/8

Feel free to merge and release a new version of `openy_features`.